modules/ROOT/pages/oauth2-policies-new.adoc

```diff
@@ -3,62 +3,27 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-// this repeats some info from external-oauth-2.0-token-validation-policy, but we need an opener page
+To secure an API in Anypoint Platform using OAuth 2.0, you can use the following policies and the matching OAuth 2.0 provider:
 
-To secure an API in Anypoint Platform using OAuth, use one of the following policies and the matching OAuth provider instead of the OAuth 2.0 Access Token Enforcement Using External Provider Policy and a Mule Provider:
+* One of the following policies available in federated organizations, are configured with specific OAuth 2.0 providers:
```

Comment:
```suggestion
* One of the following policies available in federated organizations configured with specific OAuth 2.0 providers:
```

Reply:
Note that you have init. capped all policy names below. GLOBALLY search for "policy" and standardize.

Reply:
Policy names are to be init caps excluding the word "policy". That's what you asked me to follow.